static/js/panels.js

```diff
@@ -78,8 +79,23 @@ export function show_profile_incomplete(is_profile_incomplete) {
     }
 }
 
+export function is_timezone_inconsistent() {
```

Comment:
I think it's better to create `is_timezone_consistant`  function than `is_timezone_inconsistent` function. It's more easy to read.